Platform/AlderlakeBoardPkg/Library/Stage1ABoardInitLib/Stage1ABoardInitLib.c

```diff
@@ -119,6 +121,73 @@ EarlyPlatformDataCheck (
   }
 }
 
+/**
+  CAR created by ACM covers only Initial code region.
+  This function will expand CAR to cover full flash to improve performance.
```

Comment:
Maybe update this function description. This function enables code cache for BIOS region in flash (PcdFlashSize is BIOS region size). CAR  (Cache As Ram) means a part of cache is used as RAM which is readable and writable. This function is not to set or expand a CAR region (It is already setup by FSPT).